ui/src/statistics/spurious-failures-view.js

```diff
@@ -20,10 +20,13 @@ const SpuriousFailuresView = props => {
 
   const [results, setResults] = useState([]);
 
+  const [loading, setLoading] = useState(true);
+
   const fetchStatistics = params => {
     fetch('/api/tests/statistics/spurious-failures/' + '?' + new URLSearchParams(params))
       .then(response => response.json())
       .then(result => {
+        setLoading(false);
```

Comment:
This might be a bit better in a `.finally()` after the last `.then()` to prevent the loading indicator to show indefinitely...

Reply:
Yeh, good idea! adding